docs/src/components/search.js

```diff
@@ -85,15 +97,32 @@ export default class Search extends Component {
       }
     }
 
-    this.props.onSearch(query)
     this.index = this.getOrCreateIndex()
+      // Query the index with search string to get an [] of IDs
+    const results = this.index
+      .search(query, searchOptions)
+      // Map over each ID and return the full document
+      .map(({ ref }) => this.index.documentStore.getDoc(ref))
+
+    let height = 0
+    let visibleResultsCount
+    // Calculate maximum count of results that fits in current viewport height
+    for(const [index, doc] of results.entries()) {
+      // 54, 26 are the height of search results item.
+      // They are unknonw until rendered. Is there a better way to set these values?
+      if(doc.type === 'API method') height += 54
```

Comment:
There probably is a more dynamic way to do this, but if you end up keeping this implementation I would suggest changing these [magic numbers](https://en.wikipedia.org/wiki/Magic_number_%28programming%29) into `const` declarations.

Reply:
I think it gets bit a complicated with dynamic way so I just end up keeping numeric values. Please check commits below.